nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py

```diff
@@ -422,16 +438,11 @@ def create_train_dataloader(cls, config):
         )
 
     @classmethod
-    def create_validation_dataloader(cls, config):
+    def create_validation_dataloader(cls, dataset, config):
```

Comment:
This is making it more difficult to write a test scripts that run validation. Previously we could say `loader = experiment_class.create_validation_loader(config)`. Now we have to 
figure out how to instantiate our own dataset.

Reply:
This is not neccessarily true. There's a new class method called as `experment_class.create_loaders` that will return train_loader and val_loader (and test_loader if any).